src/modules/UI/scenes/SendConfirmation/__snapshots__/sendConfirmation.test.js.snap

```diff
@@ -226,7 +235,7 @@ exports[`SendConfirmation should render with standard props 1`] = `
           resetSlider={false}
           sliderDisabled={true}
         />
-      </View>
+      </Footer>
```

Comment:
I'll be honest, this snapshot stuff looks like whole heck of a lot of additional overhead...